#This patch file is from Qualcomm Innovation Center, Inc. and is provided under the following license:
#
#Copyright (c) 2025 Qualcomm Innovation Center, Inc. All rights reserved.
#SPDX-License-Identifier: BSD-3-Clause-Clear
From 9bcc2571830e4f56a353887139689d65105a66d6 Mon Sep 17 00:00:00 2001
From: Jiaxing Shi <quic_jiaxshi@quicinc.com>
Date: Thu, 20 Feb 2025 17:34:09 +0800
Subject: [PATCH] {jazzy}: set ROS_UNRESOLVED_DEP-rapidjson-dev to rapidjson

This is a temporary patch. The unresolved dependency was updated in rosdistro
repository. It will be fixed in next update of generated recipes.

Upstream-Status: Pending

Signed-off-by: Jiaxing Shi <quic_jiaxshi@quicinc.com>
---
 .../include/jazzy/generated/superflore-ros-distro.inc           | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meta-ros2-jazzy/conf/ros-distro/include/jazzy/generated/superflore-ros-distro.inc b/meta-ros2-jazzy/conf/ros-distro/include/jazzy/generated/superflore-ros-distro.inc
index 467325fad..dc21184f0 100644
--- a/meta-ros2-jazzy/conf/ros-distro/include/jazzy/generated/superflore-ros-distro.inc
+++ b/meta-ros2-jazzy/conf/ros-distro/include/jazzy/generated/superflore-ros-distro.inc
@@ -5348,7 +5348,7 @@ ROS_UNRESOLVED_DEP-qml-module-qtquick-window2 = "ROS_UNRESOLVED_DEP-qml-module-q
 ROS_UNRESOLVED_DEP-qml-module-qtquick2 = "ROS_UNRESOLVED_DEP-qml-module-qtquick2"
 ROS_UNRESOLVED_DEP-qtquickcontrols2-5-dev = "ROS_UNRESOLVED_DEP-qtquickcontrols2-5-dev"
 ROS_UNRESOLVED_DEP-range-v3 = "ROS_UNRESOLVED_DEP-range-v3"
-ROS_UNRESOLVED_DEP-rapidjson-dev = "ROS_UNRESOLVED_DEP-rapidjson-dev"
+ROS_UNRESOLVED_DEP-rapidjson-dev = "rapidjson"
 ROS_UNRESOLVED_DEP-ros-ign-bridge = "ROS_UNRESOLVED_DEP-ros-ign-bridge"
 ROS_UNRESOLVED_DEP-ros-ign-gazebo = "ROS_UNRESOLVED_DEP-ros-ign-gazebo"
 ROS_UNRESOLVED_DEP-rti-connext-dds-6.0.1 = "ROS_UNRESOLVED_DEP-rti-connext-dds-6.0.1"
-- 
2.34.1

